src/gui/MainWindow.cpp

```diff
@@ -176,21 +176,30 @@ void MainWindow::processCapture(const CaptureDto &capture)
 		return;
 	}
 
-	showImage(capture);
+	loadImage(capture);
+
+	if (mConfig->autoShowMainWindowEnabled()) {
+		showWindow();
+	}
```

Comment:
If you can think of a good name for a method for this `if`, you could pull it out in its own method, might improve readability of the code. Something like `showMainWindowAfterScreenshot` or obviously something better.

Reply:
How about`showMainWindowIfRequired` ?

Reply:
Sounds good.